src/frontend/src/utils/http-request.util.ts

```diff
@@ -22,7 +22,9 @@ export class HttpRequest {
     }
 
     if (config.headers?.['Content-Type'] === 'application/json') {
-      return response.json();
+      const text = await response.text();
```

Comment:
this part is to process empty response from backend, the logout endpoint will return empty response and will throw an error if we direct call response.json() func

Reply:
@binggaofw  - if the expected content type is `application/json` -> do we still anticipate text or empty text here?

Reply:
Try using empty responses, like null - instead of empty string - and see if it fits your use case?